sqlitecluster/SQLite.cpp

```diff
@@ -284,6 +285,20 @@ void SQLite::_sqliteLogCallback(void* pArg, int iErrCode, const char* zMsg) {
         // 17 is the length of "conflict at page" and the following space.
         const char* offset = strstr(zMsg, "conflict at page") + 17;
         _conflictPage = atol(offset);
+
+        // 17 is the length of "part of db table" and the following space.
+        const char* tableOffset = strstr(zMsg, "part of db table") + 17;
+
+        // Check if the tableOffset exists since not all conflicts are on tables
+        if (tableOffset) {
+            // Based on the SQLite log line, we should always have ';' after the table name,
+            // so let's use it to finish this loop.
+            int i = 0;
+            while (tableOffset[i] != ';') {
+                _conflictTable += tableOffset[i];
+                i++;
+            }
```

Comment:
Instead of this loop, just call `strstr` again, which will check that we actually see the `;`.

Then you can set `_conflictTable` with constructor #8 here: https://en.cppreference.com/w/cpp/string/basic_string/basic_string by doing something like:
```
_conflictTable = string(tableOffset, semicolonOffset - tableOffset)
```

(might need a ` - 1` at the end?)

Reply:
Thanks for the suggestion! Changed, and therés no need for the `-1`: 
![image](https://github.com/user-attachments/assets/5e844924-4ab4-4609-86d3-02aa0c1a564d)